docs/HowToManageDeviceModel.md

```diff
@@ -0,0 +1,26 @@
+# How to Import / Export device Model.
```

Comment:
1. Could you check this file on markdown view? It looks not good. You need to change a line between picture and sentence.
2. This is content of SmartThings. But I can't know SmartThings doc from title.